packages/strapi-generate-new/lib/resources/files/config/server.js

```diff
@@ -0,0 +1,13 @@
+module.exports = ({ env }) => ({
+  host: env('HOST', '0.0.0.0'),
+  port: env.int('PORT', 1337),
+  proxy: {
+    enabled: false,
+  },
+  cron: {
+    enabled: false,
+  },
+  admin: {
+    autoOpen: false,
```

Comment:
Maybe consider using `env()` with a default value to keep consistency with `host` & `port`? (+ it would encourage the use of `env()` for this kind of configuration)


_example:_
```javascript
  proxy: {
    enabled: env.bool('PROXY_ENABLED', false),
  },
  cron: {
    enabled: env.bool('CRON_ENABLED', false),
  },
  admin: {
    autoOpen: env.bool('ADMIN_AUTO_OPEN', false),
  },
```

Reply:
Using env vars for every variables isn't really a best practice as it adds complexity when it isn't required (e.g if you don't use cron in your app you will never need to enable it in production). It will make things harder for us to document.